File: Iniciante/1042 - Sort Simples/C#/1042 - Sort Simples.cs
using System;

class URI {
    static void Main() {
        int aux;
        // separa os valores para cada posição do array
        string[] num = Console.ReadLine().Split(' ');
            int X = Int32.Parse(num[0]);
            int Y = Int32.Parse(num[1]);
            int Z = Int32.Parse(num[2]);

            if(X > Y) { // ordena os valores em ordem crescente
                aux = X;
                X = Y;
                Y = aux;
            }
            if(X > Z) {
                aux = X;
                X = Z;
                Z = aux;
            }

            if(Y > Z) {
                aux = Y;
                Y = Z;
                Z = aux;
            }

            Console.WriteLine("{0}\n{1}\n{2}", X, Y, Z); // saída
            Console.WriteLine("\n{0}\n{1}\n{2}", num[0], num[1], num[2]);
    }
}

File: Iniciante/1042 - Sort Simples/C/1042 - Sort Simples.c
#include <stdio.h>

int main() {
	int A, B, C; // variáveis
	int X, Y, Z, aux;
	
	scanf("%d %d %d", &A, &B, &C); // entrada
	
	X = A;
	Y = B; // atribuição de valores
	Z = C;
	
	if(X > Y) { // ordenação dos valores
		aux = X;
		X = Y;
		Y = aux;
	}
	if(X > Z) {
		aux = X;
		X = Z;
		Z = aux;
	}
	
	if(Y > Z) {
		aux = Y;
		Y = Z;
		Z = aux;
	}

	printf("%d\n%d\n%d\n\n", X, Y, Z);
	printf("%d\n%d\n%d\n", A, B, C); // saída
	
	return 0;
}